crates/solvers/openapi.yml

```diff
@@ -569,11 +607,6 @@ components:
                 "sellToken" for sell orders, and "buyToken" for buy orders.
               allOf:
                 - $ref: "#/components/schemas/TokenAmount"
```

Comment:
The `executedAmount` also appears to be outdated.
It's actually part of `JitTrade` and `Fulfillment` and should probably not be listed on the `Trade`. Also it's not a `TokenAmount` but a simple `U256`.

Reply:
Since JitTrade and Fulfillment are flattened this actually checks out (cf the example responses)
<img width="572" alt="image" src="https://github.com/cowprotocol/services/assets/1200333/3280b46c-a4c1-4bab-a432-fbf606639c9d">


Will still put it on each type. `TokenAmount` is just a placeholder for U256 though.